.buildkite/pipelines/pull_request/base.yml

```
@@ -87,7 +87,7 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 40
-    parallelism: 2
+    parallelism: 6
```

Not sure how the parallelism works, but in the last run groups 5 and 6 are empty.  If there's something I'm missing feel free to leave it.
```suggestion
    parallelism: 4
```